HLTrigger/Phase2/python/HLT_75e33/modules/L1TkMuons_cfi.py

```diff
@@ -0,0 +1,49 @@
+import FWCore.ParameterSet.Config as cms
+
+L1TkMuons = cms.EDProducer("L1TkMuonProducer",
+    CHI2MAX = cms.double(100.0),
+    DRmax = cms.double(0.5),
+    ETABARRELOVERLAP = cms.double(0.83),
+    ETAMAX = cms.double(5.0),
+    ETAMIN = cms.double(0),
+    ETAOVERLAPENDCAP = cms.double(1.24),
+    L1BMTFInputTag = cms.InputTag("simKBmtfDigis","BMTF"),
+    L1EMTFInputTag = cms.InputTag("simEmtfDigis","EMTF"),
+    L1EMTFTrackCollectionInputTag = cms.InputTag("simEmtfDigis"),
+    L1OMTFInputTag = cms.InputTag("simOmtfDigis","OMTF"),
+    L1TrackInputTag = cms.InputTag("TTTracksFromTrackletEmulation","Level1TTTracks","RECO"),
```

Comment:
@fwyzard I think this process label could be removed, too.

Reply:
Thanks, should be fixed now.